### PR TITLE
Modified about us page to the current version of EvalAI

### PR DESCRIPTION
**Pull request by Sanji515**
- Link to live demo: http://pr-134-evalai.surge.sh

**Old UI (Screenshot 1):**

Screenshot from 2019-06-10 17-52-55

**Old UI (Screenshot 2):**

Screenshot from 2019-06-10 17-53-11

**New UI (Screenshot 3):**

Screenshot from 2019-06-02 14-06-38

### CODE REVIEW

**Comment by Shekharrajak** [comment]
It would be better to attach screenshot of old UI as well as new changed UI (of this PR). Also please don't remove the demo url link.

**Comment by Sanji515** [comment]
> It would be better to attach screenshot of old UI as well as new changed UI (of this PR). Also please don't remove the demo url link.

@Shekharrajak done, please review

File: src/app/components/about/about.component.scss [hunk 1]
```diff
@@ -2,46 +2,80 @@
...
-        letter-spacing: 0.5px;
-      }
-    }
+  .about-content-column {
+    left: 0px;
   }
 }
 
-@include screen-small {
+@include screen-medium {
```
Comment (Shekharrajak) [inline src/app/components/about/about.component.scss:45]: Why it was required ?

File: src/app/components/about/about.component.html [hunk 2]
```diff
@@ -1,22 +1,31 @@
 <app-header-static></app-header-static>
 
-<div class="about-content container-top about-container">
-	<img src="../assets/images/about_image.png" />
```
Comment (Shekharrajak) [inline src/app/components/about/about.component.html:4]: It looks like you didn't remove this image from assets folder. Is it needed somewhere else?

Comment (Sanji515) [reply src/app/components/about/about.component.html:4]: no I don't think so, I'll remove it now :)

File: src/app/components/about/about.component.scss (hunk 1 above)
Comment (Sanji515) [reply src/app/components/about/about.component.scss:45]: @Shekharrajak I added this to remove the horizontal scroll

File: src/app/components/about/about.component.html (hunk 2 above)
Comment (Sanji515) [reply src/app/components/about/about.component.html:4]: @Shekharrajak done

**Review by lunayach — APPROVED** [review]

**Review by Shekharrajak — APPROVED** [review]

**Pull request merged**